src/verifysignature.c

```diff
@@ -0,0 +1,113 @@
+// Binary to help us debug the signature verification.
+// It works on almost all supported public key types and message digests.
+//
+// Currently the signature has to be in binary format. If it is in base64
+// or other encoding, you can convert it into binary format save it in file and
+// run this binary to validate the signature.
+//
+// TODO(amaltas): Support for base64 encoded signatures.
```

Comment:
Maybe change the name in the TODO or remove it. (Is it still applicable?)

Reply:
Done.